rites/roman1969/src/locales/pt-br.ts

```diff
@@ -497,7 +497,8 @@ export const locale: Locale = {
     sunday_of_the_word_of_god: '3º Domingo do Tempo Comum, ou Domingo da Palavra de Deus',
     sylvester_i_pope: 'São Silvestre I, papa',
     teresa_benedicta_of_the_cross_stein_virgin: 'Santa Teresa Benedita da Cruz, virgem e mártir',
-    teresa_of_jesus_of_avila_virgin: 'São João de Ávila, presbítero e doutor da Igreja',
+    // src: https://www.cnbb.org.br/liturgia-diaria/ 15-Oct-2025 (Retrieved 24-Jun-2025)
+    teresa_of_jesus_of_avila_virgin: 'Santa Teresa de Jesús (de Ávila), virgen y doctora de la Iglesia',
```

Comment:
You used a Spanish celebration nsme for some reason. Update it to this please:

```suggestion
    teresa_of_jesus_of_avila_virgin: 'Santa Teresa de Jesus, virgem e doutora da Igreja',
```